btr-web/btr-main-app/components/person-info/Citizenship.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="w-2/3 mr-2" :class="countries.length === 1 ? 'flex justify-center' : 'grid gap-x-1 grid-cols-2'">
+    <UTooltip
```

Comment:
we have BcrosTooltip already, that is wrapping and setting some style, can you use it here ?